tests/dune

```diff
@@ -1,3 +0,0 @@
-(cram
```

Comment:
Please don't remove this file.

Reply:
Oh I don't now what's happened! Good catch ;)

Reply:
In fact I'm used to remove all the dune files in tests/ when gentest has produced invalid rules. I didn't notice there is a versioning dune file in this folder now.

Reply:
I was going to say that it's not needed since #729 but actually we didn't merge that one because of a CI issue! I will take a look